### PR TITLE
update dependencies, principally raven

### PR DESCRIPTION
**Pull request by Meg4mi**
Receiving this with current plugin version : "This event was reported with an old version of the node SDK."

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
   },
   "license": "MIT",
   "dependencies": {
-    "hoek": "^4.1.1",
-    "raven": "~2.0.2"
+    "hoek": "^4.2.0",
+    "raven": "^2.2.1"
```
Comment (jsynowiec) [inline package.json]: `raven` should be pinned to a bugfix version.

File: package-lock.json [hunk 2]
```diff
@@ -1284,7 +1284,6 @@
       "requires": {
         "anymatch": "https://registry.npmjs.org/anymatch/-/anymatch-1.3.0.tgz",
         "async-each": "https://registry.npmjs.org/async-each/-/async-each-1.0.1.tgz",
-        "fsevents": "https://registry.npmjs.org/fsevents/-/fsevents-1.0.15.tgz",
```
Comment (jsynowiec) [inline package-lock.json]: Are you by any chance using Windows or Linux? If so, please update npm to [5.6.0](http://blog.npmjs.org/post/167963735925/v560-2017-11-27) and regenerate `package-lock` without removing fsevents.

**Review by jsynowiec — CHANGES_REQUESTED** [review]
Thanks for the PR. I've left some comments.

File: package-lock.json (hunk 2 above)
Comment (Meg4mi) [reply package-lock.json]: Using debian indeed.

**Comment by Meg4mi** [comment]
changes done :)

**Review by jsynowiec — APPROVED** [review]

**Pull request merged**